src/config.rs

```diff
@@ -185,20 +185,58 @@ fn extend(left: &mut HashMap<String, String>, right: HashMap<String, String>) {
 ///
 /// A vector of [`PathBuf`]s to the config files
 fn get_config_paths(filename: &str) -> Vec<PathBuf> {
-    [
+    #[allow(unused_mut)]
+    let mut paths = vec![
```

Comment:
```suggestion
    // we need this mut here, because we want to add data to the Vec depending on the OS
    // but our CI, running on Unix, doesn't see the extending the Vector
    #[allow(unused_mut)]
    let mut paths = vec![
```